src/StorageSync/StorageSync/ChangeLog.md

```diff
@@ -19,6 +19,8 @@
 -->
 ## Upcoming Release
 
+* Fixes security bug in token acquisition for MI server registration
```

Comment:
It is advisable to re-record atleast one test which is exercising this new code path.